build.pl

```diff
@@ -39,6 +39,9 @@
  verify    - rebuild the source with the binaries generated
              by the xbuild.
  tests     - run the tests.
+ runTestName - name of the nunit test to run (used with -tests)
```

Comment:
I think I'd prefer just `testname`.

You might also want to mention this needs to be the full name of the method incl. namespace